MobileWallet/Screens/AppEntry/Splash/SplashViewControllerSetup.swift

```diff
@@ -200,17 +202,34 @@ extension SplashViewController {
         disclaimerText.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Theme.shared.sizes.appSidePadding).isActive = true
         disclaimerText.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Theme.shared.sizes.appSidePadding).isActive = true
         disclaimerText.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
-        disclaimerText.topAnchor.constraint(greaterThanOrEqualTo: createWalletButton.bottomAnchor, constant: 10).isActive = true
+        disclaimerText.topAnchor.constraint(greaterThanOrEqualTo: createWalletButton.bottomAnchor, constant: 0).isActive = true
         disclaimerText.topAnchor.constraint(lessThanOrEqualTo: createWalletButton.bottomAnchor, constant: 30).isActive = true
     }
 
+    func setupRestoreButton() {
```

Comment:
Haven't looked yet, but please check if it's displaying fine on 4-inch screens.

Reply:
![Simulator Screen Shot - iPhone SE (1st generation) 13 5 - 2020-06-16 at 16 20 28](https://user-images.githubusercontent.com/60744748/84779647-5405d600-afed-11ea-828e-af7f2ed7903a.png)